.github/ISSUE_TEMPLATE/new_component_request.md

```diff
@@ -0,0 +1,36 @@
+---
+name: New component request
+about: Request a new component for tds-community
+---
+
+<!--
+  ### IMPORTANT SECURITY NOTE ###
+
+  When opening issues, be sure NOT to include any private or personal
+  information such as secrets, passwords, or any source code that involves
+  data retrieval. 
+-->
+
+## Problem Statement
+
+* As a _ , I would like to _ , so that I can _ 
```

Comment:
What could people write other than "As a developer/designer, I would like to see this component, so that I can work more efficiently"? Maybe it's more valuable to directly address the [component criteria](https://github.com/telusdigital/tds-community/blob/master/.github/CONTRIBUTING.md#community-component-criteria) in order to qualify a proposal.

Reply:
new features should be mainly customer driven. so as a customer:

i want to be able to easily see the fomating of my phone number so that i know when ive made a mistake

If no customer need exists then it helps set priority.